packages/select-input/index.tsx

```diff
@@ -3,6 +3,16 @@ import classNames from 'classnames'
 import Downshift from 'downshift'
 import s from './style.module.css'
 
+interface SelectInputProps {
+  name: string
```

Comment:
I'm actually not completely sure this should be a required prop. Normally I'd say all `input` elements need one, but since this is being used in a `type="hidden"` I think it's technically safe to omit it.